lumibot/traders/trader.py

```diff
@@ -2,10 +2,8 @@
 import os
 import signal
 import sys
-from concurrent.futures import ThreadPoolExecutor, as_completed
 
 # Overloading time.sleep to warn users against using it
```

Comment:
do we still need this comment?

Reply:
I don't remember consciously removing those 😄 
Must have been removed from the isort operation because they were unused.  Will clean up the comment.